lib/noted_web/live/note_live.ex

```diff
@@ -47,6 +47,17 @@ defmodule NotedWeb.NoteLive do
     {:noreply, assign(socket, note: note)}
   end
 
+  @impl true
+  def handle_event("remove_file_image", %{"file_image_id" => file_image_id}, socket) do
```

Comment:
I would call these "remove_image" and "image_id" just to keep things simple.

Reply:
Or if you are using it for both file and image. Just `remove_file` and `file_id`.